app/src/main/java/agersant/polaris/PolarisPlaybackService.java

```diff
@@ -53,10 +56,43 @@
 	private Handler autoSaveHandler;
 	private Runnable autoSaveRunnable;
 
+	private Runnable dataUpdateRunnable;
+	private Handler dataUpdateHandler;
```

Comment:
Can you rename these two to mediaSessionUpdateRunnable/Handler so it's a bit clearer what they are for?